### PR TITLE
Update cassandra.md

### PR DESCRIPTION
**Pull request by Sajotim**
Fix format errors

### CODE REVIEW

**Comment by chenrui333** [comment]
can you attach some screenshots? or it is just some coding style fix.

**Comment by Sajotim** [comment]
image
image
And similar errors below

File: content/zh/docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -214,19 +214,19 @@ kubectl apply -f cassandra-statefulset.yaml
 
 1.获取 Cassandra StatefulSet:
```
Comment (howieyuen) [inline content/zh/docs/tutorials/stateful-application/cassandra.md:215]: good catch!
the sorted list should have a blank symbol after the dot, please modify these by the way~
Thanks 🍭

Comment (Sajotim) [reply content/zh/docs/tutorials/stateful-application/cassandra.md]: Amended in the latest commit

Comment (howieyuen) [reply content/zh/docs/tutorials/stateful-application/cassandra.md]: please squash commits into one 😄

**Comment by tengqm** [comment]
This page has quite some indentation problems. Please try do a thorough fix?

**Pull request closed without merging**